### PR TITLE
add chaos mesh proposal  for sandbox entry

### PR DESCRIPTION
**Pull request by dcalvin**
Signed-off-by: Calvin Weng <wenghao@pingcap.com>

In this proposal, we would like to propose Chaos Mesh as a CNCF sandbox project. Chaos Mesh is a Chaos Engineering platform that orchestrates chaos experiments on Kubernetes environments. See more in the proposal. Thanks.

### CODE REVIEW

**Comment by dcalvin** [comment]
Thanks @caniszczyk for the update. Could you please let us know the time frame for the next step, e.g, SIG presentation by my understanding?

File: proposals/chaos-mesh.md [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+
+**Existing sponsorship**
+
+PingCAP
+
+**Statement on alignment with CNCF charter mission**
+
+Our team believes Chaos Mesh will be a great fit for CNCF. As manifested in the CNCF mission:
+
+_““These techniques enable loosely coupled systems that are **resilient, manageable, and observable**. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil. “_
```
Comment (dcalvin) [inline proposals/chaos-mesh.md]:
```suggestion
_“These techniques enable loosely coupled systems that are **resilient, manageable, and observable**. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil. “_
```

**Comment by amye** [comment]
@cncf/toc: Can we get a recommendation for SIG Review?
@dcalvin, @queenypingcap - that's the step we're waiting on.

**Comment by brendandburns** [comment]
This should go to SIG-Networking, I think.

**Comment by dcalvin** [comment]
> This should go to SIG-Networking, I think.

Thanks @brendandburns. I remember @caniszczyk also recommended SIG-APP-Delivery, that's where we created an issue in the first place (https://github.com/cncf/sig-app-delivery/issues/23). Could you also confirm, Chirs?

**Comment by michelleN** [comment]
I agree with Brendan's assessment of this falling under SIG Networking.

@leecalcote - can you confirm this project is within the scope of your SIG? If so, could also schedule this project for a presentation/review and come back to us with a recommendation?

**Comment by leecalcote** [comment]
Yes, @michelleN, @brendandburns. And warmly welcomed to boot, @dcalvin.

**Comment by dcalvin** [comment]
> Yes, @michelleN, @brendandburns. And warmly welcomed to boot, @dcalvin.

Great! Thanks all for confirming and the welcome. @leecalcote Could we reserve a slot in the sig meeting for presentation?

**Comment by leecalcote** [comment]
@dcalvin, yes, we have you on today's agenda.

**Comment by dcalvin** [comment]
Hi, @leecalcote @amye How are we going with SIG review? Is there something you need from our end?

**Comment by queenypingcap** [comment]
Hi @leecalcote , how is it going with SIG review? Is there anything you need from us?

**Comment by leecalcote** [comment]
Hi @Queeny, the review you have given of the draft PR is helpful. This feedback needs to be incorporated and the review just needs tied off today for completion.

File: proposals/chaos-mesh.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+GitHub
+
+Slack channel: [#sig-chaos-mesh](https://tidbcommunity.slack.com/archives/CS7U0QV8E)
+
+Community meetings (Planing)
+
+**Adopters or potential users?**
+
+TiKV/TiDB projects   
+Netease (testing)
```
Comment (dcalvin) [inline proposals/chaos-mesh.md:381]:
```suggestion
- [Celo](https://celo.org/)
- [Dailymotion](https://www.dailymotion.com/)
- [NetEase Fuxi Lab](https://www.crunchbase.com/organization/netease-fuxi-lab)
- [JuiceFS](http://juicefs.com/?hl=en)
- [Meituan-Dianping](https://about.meituan.com/en)
- [PingCAP](www.pingcap.com)
- [Xpeng Motors](https://en.xiaopeng.com/)
```

**Comment by caniszczyk** [comment]
This was accepted into the Sandbox from today's TOC meeting to go over the sandbox backlog.

Project onboarding will be here: #493

**Comment by leecalcote** [comment]
Whoo-hoo!

**Pull request merged**